requirements.txt

```diff
@@ -1,4 +1,4 @@
-velociraptor>=0.15.8
+velociraptor>=0.16.0
```

Comment:
This version of the velociraptor python library supports SOAP; this is why I updated this line of code.